### ISSUE TITLE
Blurred (obscured) feedback is read by screen reader even though they are supposed to be not available to view

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1318]
#A11y #accessibility issue
**Describe the bug**
Install the Retrospectives Extension:

Go to the Azure DevOps Marketplace ([Extensions for Visual Studio family of products | Visual Studio Marketplace](https://marketplace.visualstudio.com/azuredevops/)) and search for the "Retrospectives" extension.
Click "Get it free" and follow the prompts to install it to your Azure DevOps organization.
Create a Retrospective:

Navigate to your Azure DevOps project.
In the left-hand menu, select "Boards" and then "Retrospectives."
Click on "New Retrospective" to create a new session.
Enable Obscure Feedback:

In the retrospective settings, look for the option labeled "Obscure Feedback."
Enable this setting to obscure feedback until the collection phase is complete.

Add some feedback items under each column by you and another user
Navigate around to see if the blurred (Obscured) feedback is announced by screen readers.

Actual result:
The obscured feedback is announced by screen reader, even though they cannot be read by sighted users.

<img width="734" height="406" alt="Image" src="https://github.com/user-attachments/assets/f99a3954-2385-4d0f-8b22-8625bcd6d920" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/7cb5bf53-7b8f-486c-99d1-b4693f72ed86" />
